### ISSUE TITLE
Never ending record updates

### ISSUE THREAD

**Issue by electerious** [Burnsy/gatsby-plugin-ackee-tracker#3]
There might be an issue as far as I can tell (without knowing how gatsby works):

When `onRouteUpdate` runs on every route change than `ackee-tracker` never stops to update the duration of old records. When you call `instance.record()`, Ackee creates a new record on the server and updates the record constantly to track the duration of the visit.

There's currently no way to stop the record updates, so this is also an issue on my side and I need to fix it first.

**Comment by BurnedChris** [comment]
Hello, 

How it basically works is that i have attached the ackee-tracker to onRouteUpdate, That is when a user changes page, so it would then swap from /a to /b  and re-run the instance.record(). 

It might be better if runs on onClientEntry (Called when the Gatsby browser runtime first starts). But i am unsure if it would then track different pages or just the first one. As i not looked too much into how the tracker defines what page it is on.

**Comment by electerious** [comment]
I've updated `ackee-tracker`. The update of old records will now be canceled when you call `record()` once again: https://github.com/electerious/ackee-tracker/blob/master/CHANGELOG.md#322---2019-11-03

It's just important to use the same instance instead of recreating one. I recommend to create the `ackee-tracker` instance [`onInitialClientRender`](https://www.gatsbyjs.org/docs/browser-apis/#onInitialClientRender) and then call `instance.record()` [`onRouteUpdate`](https://www.gatsbyjs.org/docs/browser-apis/#onRouteUpdate).

Let me know if there's anything I can help with. And thanks again for creating the plugin!

**Comment by BurnedChris** [comment]
I just spent a few hours fixing this and it now has been split into two  onInitialClientRender and instance.record().

The only gotcha i found was that i needed to add .record() after it created the tracker so it would track the first page! I released some busted versions by accident. But latest version 1.1.1 has that fixed! 

If you just want to look through it before i close this issue! i would be happy to take a second throw at it. Having it as a function seemed like the most straight forward way in my mind to be able to keep the instance between both functions
